admin_manual/installation/examples/lets_encrypt/nginx.rst

```diff
@@ -0,0 +1,183 @@
+Setup NGINX
+===========
+
+The following is an example setup process for NGINX, please adapt it to your exact needs.
+
+NGINX ssl_dhparam
+-----------------
+
+If not already present, add an `ssl_dhparam`_ directive and a new certificate with stronger keys (which improves `forward secrecy`_). 
```

Comment:
Proposal (similar to Apache):
...with stronger keys for `Diffie-Hellmann`_ based key exchange (which...

.. _Diffie-Hellman: https://en.wikipedia.org/wiki/Diffie–Hellman_key_exchange